packages/mobile/src/components/social-button/SocialButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useMemo } from 'react'
+
+import type { ButtonProps } from 'app/components/core'
+import { Button } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+export type SocialButtonProps = ButtonProps
+
+const useStyles = makeStyles(({ spacing }) => ({
+  button: {
+    padding: spacing(3),
+    height: 64
+  },
+  text: {
+    fontSize: 20
```

Comment:
Potentially us typography? Is that not an established value or something?

Reply:
This is passing in as a style to the button, so don't think there is a way to use typography here unless we wanted to add extra props to `Button`